scripts/fetch/fetchGA.R

```diff
@@ -3,21 +3,25 @@ library(googleAnalyticsR)
 library(googleAuthR)
 library(dplyr)
 library(data.table)
-library(sbtools)
 library(assertthat)
+library(aws.s3)
+library(aws.signature)
```

Comment:
Does this need to be in the viz.yaml, or is counting on transitive dependencies in this case good enough?